plugins/alerting/pkg/alerting/management.go

```diff
@@ -118,17 +112,24 @@ func (p *Plugin) watchCortexClusterStatus() {
 		panic(err)
 	}
 	// acquire cortex client
-	var adminClient cortexadmin.CortexAdminClient
-	for {
-		ctxca, ca := context.WithTimeout(p.Ctx, 5*time.Second)
-		acquiredClient, err := p.adminClient.GetContext(ctxca)
-		ca()
-		if err != nil {
-			lg.Warn("could not acquire cortex admin client within timeout, retrying...")
-		} else {
-			adminClient = acquiredClient
-			break
+	// var adminClient cortexadmin.CortexAdminClient
+	// for {
+	// 	ctxca, ca := context.WithTimeout(p.Ctx, 5*time.Second)
+	// 	acquiredClient, err := p.adminClient.GetContext(ctxca)
+	// 	ca()
+	// 	if err != nil {
+	// 		lg.Warn("could not acquire cortex admin client within timeout, retrying...")
+	// 	} else {
+	// 		adminClient = acquiredClient
+	// 		break
+	// 	}
+	// }
+	adminClient, err := p.adminClient.GetContext(p.Ctx)
```

Comment:
I am pretty sure this won't work correctly, the previous code should've probably just blocked on `p.adminClient.Get()`

Reply:
changed to `p.adminClient.Get()`